ui/vic-ui-h5c/vic-service/src/main/java/com/vmware/vic/model/VirtualContainerHostVm.java

```diff
@@ -78,7 +86,12 @@ private void processExtraConfig(ArrayOfOptionValue ovs) {
 						}
 					}
 					_clientIp = sb.toString();
-					break;
+					continue;
+				}
+
+				if (EXTRACONFIG_DOCKER_PERSONALITY_ARGS_KEY.equals(key)) {
+				    _isUsingTls = ((String)ov.getValue()).indexOf("2376") > -1;
```

Comment:
maybe define `final String some_name = 2376` at the very beginning and make `some_name` self-explanatory.